cpp/oneapi/dal/table/BUILD

```diff
@@ -28,29 +28,29 @@ dal_test_suite(
 )
 
 dal_test_suite(
-    name = "row_accessor_tests",
+    name = "catch2_tests",
     srcs = [
         "test/row_accessor.cpp",
+        "test/homogen.cpp",
     ],
     dal_deps = [":table"],
     framework = "catch2",
 )
 
 dal_test_suite(
-    name = "interface_tests",
+    name = "gtest_tests",
```

Comment:
I am wondering why gtest are for column_accessor  while catch2 tests are for row_accessor.

Reply:
The test were initially written in gtest, I'm gradually moving them to catch2. At this iteration, I migrated homogen table tests